src/components/History.js

```diff
@@ -1,14 +1,40 @@
 /* eslint-disable class-methods-use-this */
 import React from "react";
+import { withRouter } from "react-router";
+import PropTypes from "prop-types";
+import { Link } from "react-router-dom";
+
+import dummyData from "../dummy-data.json";
+
+import StationName from "./StationName";
+import AddNewCase from "./AddNewCase";
 import Navbar from "./Navbar";
 
-export default class History extends React.Component {
+class History extends React.Component {
+  state = {
+    stations: dummyData.history
+  };
+
   render() {
+    const stationElements = this.state.stations.map((element, index) => (
+      <Link key={index} to={`/history/${element.stationName}`}>
+        <div key={index}>{element.stationName}</div>
+      </Link>
+    ));
     return (
-      <div>
-        <h2>History list of Stations</h2>
+      <React.Fragment>
+        <h1>History</h1>
+        <p>Add new</p>
+        {stationElements}
         <Navbar />
-      </div>
+      </React.Fragment>
     );
   }
 }
+
+History.propTypes = {
+  cases: PropTypes.arrayOf(PropTypes.object),
+  match: PropTypes.object
+};
+
+export default withRouter(History);
```

Comment:
Did you find any documentation about when this is needed and when not?

Reply:
If you looked at it, it would be great if you could summarise it for us both, we haven't looked at it.